@here/harp-mapview/lib/Tile.ts

```diff
@@ -492,56 +487,43 @@ export class Tile implements CachedResource {
     }
 
     /**
+     * @internal
+     * @deprecated
+     *
      * Gets the list of developer-defined [[TextElement]] in this `Tile`. This list is always
      * rendered first.
      */
     get userTextElements(): TextElementGroup {
-        return this.m_userTextElements;
+        return (
+            this.m_textElementGroups.groups.get(Number.MAX_SAFE_INTEGER) ??
+            new TextElementGroup(Number.MAX_SAFE_INTEGER)
```

Comment:
should this be added also to the this.m_textElementsGroup, otherwise calling this function multiple times will allocate multiple TextElementGroup's

Reply:
done